README.md

```diff
@@ -1,78 +1,44 @@
 # Web Archive Search Index API and UI
 
-An API wrapper to the Elasticsearch index of web archival collections and a web UI to explore those indexes.
+An API wrapper to the Elasticsearch index of web archival collections and a web UI to explore those indexes. A part of the [story-indexer stack](https://github.com/mediacloud/story-indexer). Mantained as a separate repository for future legibility. 
 
 ## ES Index
 
-The API service expects the following ES index schema, where `title` and `snippet` fields must have the `fielddata` enabled (if they have the type `text`) and `first_captured` field stores 14-digit datetime in the format `YYYYMMDDhhmmss`.
-
+The API service expects the following ES index schema, where `title` and `snippet` fields must have the `fielddata` enabled (if they have the type `text`).
+This is currently defined in the story-indexer stack, but is replicated here for convenience. 
```

Comment:
Replication seems.... less than ideal!  BUT given that they're in different repos, an easy/clean solution seems.... elusive!

Reply:
Agreed- the concerns are a little muddled. This seemed like the best way to document how what dependencies exist given how things are laid out now.